Exercise_4.py

```diff
@@ -7,13 +7,20 @@ def __init__(self, data):
         self.right = None
           
 """ Inorder traversal of a binary tree"""
-def inorder(temp): 
-  
+def inorder(temp):
+    if not temp:
+        return
+    inorder(temp.left) 
+    print(temp.key)
+    inorder(temp.right)
     
   
   
 """function to insert element in binary tree """
-def insert(temp,key): 
+def insert(temp,key):
+    while temp.right:
```

Comment:
This would put the nodes only on the right part of the tree and not the left even if the left part is empty